go/obscuronode/enclave/sql/sql.go

```diff
@@ -0,0 +1,160 @@
+package sql
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/ethdb"
+)
+
+const (
+	getQry    = `select kv.value from kv where kv.key = ?;`
+	putQry    = `insert or replace into kv values(?, ?);`
+	delQry    = `delete from kv where kv.key = ?;`
+	searchQry = `select * from kv where kv.key like ? and kv.key > ?`
```

Comment:
let's double check that the order is respected. 
I saw this in the doc:
"// Iterator iterates over a database's key/value pairs in ascending key order."

Reply:
Yeah good spot, I should have an 'order by', have added that and a test to verify the sort.